alf/experience_replayers/replay_buffer.py

```diff
@@ -57,13 +57,13 @@ def __init__(self,
                  num_environments,
                  max_length=1024,
                  device="cpu",
-                 name="ReplayBuffer"):
-        """Create a ReplayBuffer.
+                 name="RingBuffer"):
+        """Create a RingBuffer.
```

Comment:
Remove "create a ..." template in `__init__`. We should do this for all init functions sooner or later. Sphinx will combine the docstrings of class and init into one. So this template is literally uninformative.

Reply:
Done.  Will remember to do this for others.